datamanager/storage/spanner/spanner.go

```diff
@@ -48,13 +57,13 @@ func (mgr *SpannerDataMgr) BatchAddJobs(ctx context.Context, jobs []*model.JobDa
 }
 
 // BatchGetJobs pumps data that are due before certain due time
-func (mgr *SpannerDataMgr) BatchGetJobs(ctx context.Context, req []*model.JobDataReq) (jobs []*model.JobData, err error) {
+func (mgr *SpannerStorage) BatchGetJobs(ctx context.Context, req []*model.JobDataReq) (jobs []*model.JobData, err error) {
```

Comment:
I didn't see any place used this function

Reply:
It was originally designed for possible future use.
Shall we remove it for now, or just simply keep this?

Reply:
I think we can keep `BatchGetJobs` but the input parameters should be job ids. The reason is this function should be used when we want to know whether the job is in our system or not.

Reply:
This method was originally designed to check stored jobs under certain namespaces/queues.
I suppose the one you mentioned above can be implemented in another method, BatchGetJobsByIDs perhaps.